x-pack/platform/plugins/shared/onechat/server/services/runner/model_provider.ts

```diff
@@ -34,43 +33,6 @@ export type ModelProviderFactoryFn = (
   opts: Pick<CreateModelProviderOpts, 'request' | 'defaultConnectorId'>
 ) => ModelProvider;
 
-/**
- * Wraps an InferenceChatModel with tracking for LLM usage
- * @param chatModel - The chat model to wrap
- * @param connector - The connector associated with the model
- * @param trackingService - Optional tracking service for telemetry
- * @returns The wrapped chat model with tracking
- */
-function wrapChatModelWithTracking(
-  chatModel: InferenceChatModel,
-  connector: InferenceConnector,
-  trackingService?: TrackingService
-): InferenceChatModel {
```

Comment:
No longer need to do this, we just use inference callbacks for that tracking (also this wasn't tracking calls done using the inference API directly, only ones using the chat model)